integration/tests/basic/indexer/conftest.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+
+@pytest.fixture(scope="class")
+def nested_call_contracts(accounts, web3_client):
+    contract_a, _ = web3_client.deploy_and_get_contract(
+        "common/NestedCallsChecker", "0.8.12", accounts[0], contract_name="A"
+    )
+    contract_b, _ = web3_client.deploy_and_get_contract(
+        "common/NestedCallsChecker", "0.8.12", accounts[0], contract_name="B"
+    )
+    contract_c, _ = web3_client.deploy_and_get_contract(
+        "common/NestedCallsChecker", "0.8.12", accounts[0], contract_name="C"
+    )
+    yield contract_a, contract_b, contract_c
+
+
+@pytest.fixture(scope="function")
+def recursion_factory(accounts, web3_client):
```

Comment:
lets move all common contracts to common conftest files no to duplicate the fixtures

Reply:
Fixed ✅